main.js

```diff
@@ -1,37 +1,55 @@
-let One,Two,Three,Four,Five,Six,Seven,Eight,Nine,Ten;
-let index;
-let all = "All";
-let previus_count = 10;
+let channel_one,channel_two,channel_three,channel_four;
+let selected_category = "bbc-news";
+let channel_id = 0;
+let images = new Array();
+let title = new Array();
+let postdate = new Array();
+let description = new Array();
+let morenews = new Array();
+//-------------------------------------Fetching data-----------------------------------------
+function getAPIData(){
+	fetch(`https://newsapi.org/v2/everything?q=${selected_category}&from=2019-07-15&sortBy=publishedAt&apiKey=94a28e3dd8314a2cb51e81a385bb052a`)
+	.then(response => {
+		return response.json();
+	})
+	.then(response => {
+		return response.articles;
+	})
+	.then( response => {
+		for(let position=0; position<10; position++){
+			title.push(response[position].title);
+			description.push(response[position].description);
+			images.push(response[position].urlToImage);
+			postdate.push(response[position].publishedAt);
+			morenews.push(response[position].content);
+		}
+		return response;
+	})
+	.then( response => {
+		leftBlock(channel_id);
+		return response;
+	})
+	
+}
 window.onload = function() {
 	createHeader();
+	getAPIData();
 	createMain();
 	createFooter();
   };
 function getData(index){
-	One = {"Image": "BBC_World_News.png","Title": "BBC World News","Postdate": "Posted on 29 June,2019","Category": "Category:Category One","Description": "Lorem ipsum dolor sit amet,consectetur adipiscing elit,sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Quis ipsum suspendisse ultrices gravida.Risus commodo viverra maecenas accumsan lacus vel facilisis.Consectetur adipiscing elit,sed do eiusmod tempor incididunt ut labore et dolore magna aliquauis ipsum suspendisse ultrices gravidasus commodo.","id": 0};
-	
-	Two = {"Image": "FOX_News_Channel.png","Title": "Fox News","Postdate": "Posted on 29 June,2019","Category" : "Category:Category Two","Description": "Lorem ipsum dolor sit amet,consectetur adipiscing elit,sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Quis ipsum suspendisse ultrices gravida.Risus commodo viverra maecenas accumsan lacus vel facilisis.Consectetur adipiscing elit,sed do eiusmod tempor incididunt ut labore et dolore magna aliquauis ipsum suspendisse ultrices gravidasus commodo.","id": 1};
-	
-	Three = {"Image": "cnn.png","Title": "CNN","Postdate": "Posted on 29 June,2019","Category" : "Category:Category Three","Description": "Lorem ipsum dolor sit amet,consectetur adipiscing elit,sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Quis ipsum suspendisse ultrices gravida.Risus commodo viverra maecenas accumsan lacus vel facilisis.Consectetur adipiscing elit,sed do eiusmod tempor incididunt ut labore et dolore magna aliquauis ipsum suspendisse ultrices gravidasus commodo.","id": 2};
+	channel_one = {"Name": "BBC News","selected_category":"bbc-news","id": 0};

```

Comment:
please remove these many constant,
 create one object
ex: const a = {'b':'c'};
return a['b']   =result is c only

no need to create these many constant and array to achieve that functionality